docs/images.md

```diff
@@ -84,6 +84,25 @@ When using multiple writers, each writer should modify a distinct byte
 range. If two writers modify the same byte range concurrently they will
 overwrite each other data.
 
+### format
```

Comment:
Maybe this should be transfer_format to make it more clear. It is not the same as image format - when using the nbd backend, we always transfer raw data, even if the underlying image is using qcow2 format.